### ISSUE TITLE
Question about pass an extra argument into metrics 'Scorer' object

### ISSUE THREAD

**Issue by bofei5675** [automl/auto-sklearn#746]
I'm working on a project that needs me to pass a mask of label into the scorer. For example, if y is a vector (100,), then I will have a boolean mask with the same size.

After passing this mask, I found that the resampling strategy used hold-out set then this mask won't be able to match with the original vector size. I'm just wondering how can I fix this? Also, if someone can point out which part of code does the splitting, it would be definitely helpful. 

Thanks!

**Comment by mfeurer** [comment]
Unfortunately, this feature is not implemented yet. Auto-sklearn uses scikit-learn code for splitting. Does scikit-learn allow this?
